src/demo-app/datepicker/datepicker-demo.html

```diff
@@ -5,3 +5,8 @@
 
 <br>
 <div>{{selected?.toNativeDate()}}</div>
+
+<input [mdDatepicker]="dp">
+<button (click)="dp.open()">open</button>
+<button (click)="dp.open(true)">open (mobile)</button>
```

Comment:
What `true` means is kind of opaque when looking at this. Have you considered another method that makes it more clear what is happening? `openAsDialog()` ?